apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_form.ex

```diff
@@ -167,12 +167,18 @@ defmodule AndiWeb.EditLiveView.DataDictionaryForm do
   end
 
   def handle_event("file_upload", %{"file" => file, "fileType" => "text/csv"}, socket) do
-    new_changeset =
-      file
-      |> parse_csv()
-      |> DataDictionaryFormSchema.changeset_from_tuple_list(socket.assigns.dataset_id)
-
-    assign_new_schema(socket, new_changeset)
+    try do
+      new_changeset =
+        file
+        |> String.equivalent?("\n")
```

Comment:
Will this handle legitimately empty files? i.e. no whitespace no nothing.

Reply:
For empty CSV files, I was getting "\n".